public/js/utils/source-map-worker.js

```diff
@@ -27,21 +38,21 @@ function enableSourceMaps() {
   sourceMapsEnabled = true;
 }
 
-function _resolveSourceMapURL(source) {
-  if (path.isURL(source.sourceMapURL) || !source.url) {
+function _resolveSourceMapURL(source: Source) {
+  const { url = "", sourceMapURL = "" } = source;
```

Comment:
@jlongster @sankha93 I didn't realize that we needed to use the defaults to fix the source type. 👍 

This looks good